integr8ly/integreatly-operator
# bump cro versions to v0.24.0

# Description

Bump the cro version v0.24.0

## Type of change
Bump the cro version v0.24.0, cro has been update to the operator-sdk 1.2.0 and Go version used was updated to 1.16.2, CRO is now using bundle rather that csv for installation. 


## Verification
- install and upgrade rhmi/rhoam
- confirm that the v.0.24.0 version of cro is installed
![image](https://user-images.githubusercontent.com/16667688/114148145-600eab80-9911-11eb-9ebb-655f191a8be3.png)
- confirm that the redis/s3/rds are created
- uninstall and confirm removed

## Checklist
